armi/reactor/reactors.py

```diff
@@ -2316,3 +2324,41 @@ def processLoading(self, cs, dbLoad: bool = False):
         self.p.maxAssemNum = self.getMaxParam("assemNum")
 
         getPluginManagerOrFail().hook.onProcessCoreLoading(core=self, cs=cs)
+
+    def _axiallyExpandAssems(
+        self, assems: list, dbLoad: bool, expTag: str, detAxExp: bool
+    ):
+        """expand assembliues"""
+        axialExpChngr = AxialExpansionChanger(detAxExp)
+        runLog.header(
+            f"=========== Axially expanding {expTag} assemblies (except control) from Tinput to Thot ==========="
```

Comment:
optional: consider moving this header outside the function call. i know its duplicated code, but sending a string in to be printed right is not great. it will also help the reader to understand whats happing as they read code.

Reply:
I don't know, I'm not sure it will make a big difference to me, to save that one method argument.  #shrug

Reply:
Change made
https://github.com/terrapower/armi/pull/823/commits/de07353fcb87ce30d7af984de9d00e4199f1a408